hdr/sqlite_modern_cpp/type_wrapper.h

```diff
@@ -166,6 +166,26 @@ namespace sqlite {
 	inline void store_result_in_db(sqlite3_context* db, std::nullptr_t) {
 		sqlite3_result_null(db);
 	}
+#ifdef MODERN_SQLITE_STD_VARIANT_SUPPORT
+	template<>
+	struct has_sqlite_type<std::monostate, SQLITE_NULL, void> : std::true_type {};
+  
+	inline std::monostate get_col_from_db(sqlite3_stmt* stmt, int inx, result_type<std::monostate>) {
+    if(sqlite3_column_type(stmt,inx) != SQLITE_NULL) throw 1;
```

Comment:
@zauguin This doesn't look right to me, should we just return `std::monostate` all the time in `get_col_from_db(sqlite3_stmt*, in, result_type<std::monostate>` ?

Reply:
I wasn't sure about whether or not this was necessary so this is more from making sure that I could figure out what'd gone wrong.

I've removed it and found no problems, so I'm inclined to commit that version instead.

Reply:
@zauguin Is it good to merge? you know more than me about this part.